spark-common/src/main/resources/com/k15t/spark/content_iframe_wrapper.vm

```diff
@@ -1,20 +1,21 @@
 ## appending WithHtml to variable name instructs Velocity not to add html escaping (like & -> &amp;)
 ## which would otherwise broke the js embedded this way pretty badly...
-#set($iframeResizerJsWithHtml = $iframeResizerJs)
+#set($sparkJsWithHtml = $sparkJs)
 <script>
-    ${iframeResizerJsWithHtml}
+    ${sparkJsWithHtml}
```

Comment:
Can we merge this with the script tag below?
There's no reason why we would **want** it to be separate I guess.